aliyun/src/test/java/org/apache/iceberg/aliyun/oss/TestOSSURI.java

```diff
@@ -49,47 +49,46 @@ public void testEncodedString() {
 
   @Test
   public void invalidBucket() {
```

Comment:
we can omit the description in all of the changes in this class

Reply:
done.